### PR TITLE
added annotator analyzer cronjob

### PR DESCRIPTION
**Pull request by Sumered**

### CODE REVIEW

**Review by turowicz — APPROVED** [review]

File: helm-chart/templates/analytics/analyzer/storage.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  labels:
+    {{- include "cvat.labels" . | nindent 4 }}
+    app: cvat-app
+    tier: backend
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
```
Comment (bknulz) [inline helm-chart/templates/analytics/analyzer/storage.yml]: Add as param

Comment (bknulz) [reply helm-chart/templates/analytics/analyzer/storage.yml]: define in values

File: helm-chart/templates/analytics/analyzer/storage.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  labels:
+    {{- include "cvat.labels" . | nindent 4 }}
+    app: cvat-app
+    tier: backend
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: .Values.analytics.job.memory
```
Comment (bknulz) [inline helm-chart/templates/analytics/analyzer/storage.yml]: missing {{  }}

try dry-run to debug

**Review by kbegiedza — CHANGES_REQUESTED** [review]

**Review by kbegiedza — APPROVED** [review]
LGTM

**Comment by turowicz** [comment]
> LGTM

Lets Go To Moon

**Pull request merged**